mlos_bench/mlos_bench/tests/config/__init__.py

```diff
@@ -28,8 +37,8 @@ def locate_config_examples(config_examples_dir: str) -> List[str]:
     """
     assert os.path.isdir(config_examples_dir)
     config_examples = []
-    for root, _, files in os.walk(config_examples_dir):
-        for file in files:
+    for root, _, _files in os.walk(config_examples_dir):
+        for file in _files:
             if file.endswith(".json") or file.endswith(".jsonc"):
                 config_examples.append(path_join(root, file))
```

Comment:
```suggestion
    config_examples.extend(glob.glob("**/*.json*", root_dir=config_examples_dir, recursive=True))
```
would that work?

Reply:
Probably?  I didn't really change this logic, just the variable names due to the `files` name conflict.
I can give it a shot though.

Reply:
Nm.  `root_dir` option doesn't exist in older versions of python